lib/common.sh

```diff
@@ -267,7 +268,11 @@ setGoVersionFromEnvironment() {
 }
 
 determineTool() {
-    if [ -f "${depTOML}" ]; then
+    if [ -f "${goMOD}" ]; then
```

Comment:
Do we also need to detect if `GO111MODULE=on` to enable gomodules?

Reply:
I'd rather not until and issue is raised or we understand better the circumstances around needing to force it on.